### DISCUSSION TITLE
Support me, I want draw polyline same image

### DISCUSSION THREAD

**Discussion by CuongSofi** [Leaflet/Leaflet#9549] [Q&A]
### Checklist

- [x] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [x] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [x] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

Current Polygon  ![Image](https://github.com/user-attachments/assets/87fcec02-6412-47fb-96c8-14ff5ed062ae)

### Suggested solution

I want polyline have drawn
![Image](https://github.com/user-attachments/assets/ed76156c-ccd3-472a-a2eb-775575167d86)
Thanks.

**Comment by haneen-9** [comment]
![Capture](https://github.com/user-attachments/assets/05a2e217-b053-4934-8bfa-50d35b042bd5)

**Comment by haneen-9** [comment]
![Capture JPG2](https://github.com/user-attachments/assets/1c910a61-44fe-4be3-92ce-827a9dd29810)

**Comment by Falke-Design** [comment]
Please write full senteces and show us what you have already tried. The tutorials can help you: https://leafletjs.com/examples.html
